app/controllers/users_controller.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class UsersController < ApplicationController
+  def new
+    @user = User.new
+  end
+
+  def create
```

Comment:
This method is giving an error. You should follow rails  convention and create a user_params method
and then pass this method to `User.new `method